File: sdkwork-backend-sdk-go/types/plus_app_form.go
package types

// Application creation form
type PlusAppForm struct {
	Name string `json:"name"`
	Icon ImageMediaResource `json:"icon"`
	ResourceList AssetMediaResourceList `json:"resourceList"`
	ProjectId int `json:"projectId"`
	Description string `json:"description"`
	Version string `json:"version"`
	IconUrl string `json:"iconUrl"`
	AccessUrl string `json:"accessUrl"`
	Config AppConfig `json:"config"`
	Status string `json:"status"`
	AppType string `json:"appType"`
	Platforms AppPlatforms `json:"platforms"`
	PackageName string `json:"packageName"`
	BundleId string `json:"bundleId"`
	StoreUrl string `json:"storeUrl"`
	DownloadUrl string `json:"downloadUrl"`
}

File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_app_form.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusAppForm:
    """Application creation form"""
    name: str
    icon: ImageMediaResource = None
    resource_list: AssetMediaResourceList = None
    project_id: int = None
    description: str = None
    version: str = None
    icon_url: str = None
    access_url: str = None
    config: AppConfig = None
    status: str
    app_type: str
    platforms: AppPlatforms
    package_name: str = None
    bundle_id: str = None
    store_url: str = None
    download_url: str = None
